src/avatar.tsx

```diff
@@ -0,0 +1,120 @@
+// https://www.figma.com/file/U4ipIXOk64bdM5tSvaqPKS/%5BREADY%5D--Avatar?node-id=2%3A61
+import * as React from 'react';
+import Badge from './badge';
+import IconUserAccountRegular from './generated/mistica-icons/icon-user-account-regular';
+import {useTheme} from './hooks';
+import {createUseStyles} from './jss';
+import {useIsInverseVariant} from './theme-variant-context';
+
+import type {IconProps} from './utils/types';
+
+const useStyles = createUseStyles(() => ({
+    avatar: {
+        borderRadius: '50%',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        color: ({textColor}) => textColor,
+        background: ({backgroundColor}) => backgroundColor,
+        width: ({size}) => size,
+        height: ({size}) => size,
+        overflow: 'hidden',
+    },
+    image: {
+        width: ({size}) => size,
+        height: ({size}) => size,
+        objectFit: 'cover',
+    },
+}));
+
+/**
+ * Returns a right/top distance for the badge.
+ * The badge will be placed over the avatar's edge
+ * This is calculated using the `radius * (1 - sin(45deg))` distance minus a
+ * constant offset that depends on the badge size
+ */
+const getBadgeDistance = (size: number, badge?: boolean | number): number => {
+    if (!badge) {
+        return 0;
+    }
+    const radius = size / 2;
+    const badgeOffset = badge === true ? 5 : 10; // badge=true renders a small circle
+    return radius * (1 - Math.sin(Math.PI / 4)) - badgeOffset;
+};
+
+type AvatarProps = {
+    size: number;
+    url?: string;
```

Comment:
Looking for parity

`url` or `src` or `backgroundImage`?

<Image> use `src`
<Circle> use `backgroundImage`

Reply:
perhaps `src` so it follows the same naming as the Image component